lib/redis_memo.rb

```diff
@@ -94,6 +96,29 @@ def self.without_memo
     Thread.current[THREAD_KEY_WITHOUT_MEMO] = prev_value
   end
 
+  def self.with_max_connection_attempts(max_attempts)
```

Comment:
is this meant to be a public method?

if so, let's we add [rdoc](https://devhints.io/rdoc) for it?